VideoLocker/src/androidTest/java/org/edx/mobile/test/model/CourseOutlineTests.java

```diff
@@ -0,0 +1,25 @@
+package org.edx.mobile.test.model;
+
+import org.edx.mobile.model.ICourse;
+import org.edx.mobile.model.ISequential;
+import org.edx.mobile.model.mocked.MockedCourseOutlineProvider;
+import org.edx.mobile.services.CourseManager;
+import org.edx.mobile.test.BaseTestCase;
+
+/**
+ * Created by hanning on 4/29/15.
+ */
+public class CourseOutlineTests extends BaseTestCase {
+
+    public void testGetDisplayVersionName() throws Exception {
```

Comment:
@aleffert  renamed to CourseManagerTest. it should also be junit test.  as we just added junit test yesterday, i can move it to folder for unit test.